dev-tools/packer/.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: required
```

Comment:
It would be nice if we find a way to tell travis about subdirectories. But couldn't find a solution yet.